english/texas/texas.md

```diff
@@ -0,0 +1,4 @@
+The man informs you that you are in Texas!!
+No wonder everything is so big.
+Time to follow your [destiny](../destiny.destiny.md).
+Or just [alchemy](../alchemy/alchemy.md) time!
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.